### ISSUE TITLE
Inconsistent test runs depending on the installed ember-cli version

### ISSUE THREAD

**Issue by Windvis** [NullVoxPopuli/ember-apply#450]
I wanted to bump the embroider packages to v2 but noticed that the tests where failing locally before I even made a change.

It seems that during the tests a new ember app is generated, modified and then compared to the snapshots. It uses your globally installed ember-cli for that, so if the version doesn't match with the one that was used for generating the snapshots the tests will probably fail. This also means that the tests always fail if you don't have ember-cli installed globally. CI potentially has the same issue since the latest ember-cli version is installed globally (which might differ from the ones used for the snapshots).

Would it maybe be better to install ember-cli as a devDependency so the test runs are consistent? We can then update the snapshots when bumping the ember-cli version.

**Comment by Windvis** [comment]
I believe I gave this a try but I ran into the issue where ember-cli assumes a project is an Ember app (or addon) if ember-cli is a dependency.. So it refused to run the project or addon generators. Maybe we could just use `npx` (or the pnpm equivalent) based commands instead, with a version that's hardcoded? `npx ember-cli@some-version new ...`

It has the downside that we can't use a bot to update the version, but updating probably needs to be done by hand anyways since the snapshots will most likely not match anymore.
